solutions/bronze/cf-1808B.mdx

```diff
@@ -0,0 +1,71 @@
+---
+id: cf-1808B
+source: CF 1808B
+title: Playing in a Casino
+author: Amit Singh Bisht
+---
+
+## Video Solution
+
+Note: The video solution might not be the same as other solutions. Code in C++.
+<Youtube id="FCdL47-AyZs" />
+
+## Explanation
+
+Essentially, the solution is to calculate the difference between the sum of the larger elements and the sum of the smaller
+elements in each column, and weighting this difference by the number of times that the smaller element appears in that column.
+This is based on the observation that if we have n numbers, and we want to minimize the sum of absolute differences between
+each number and some target value, then the target value should be the median of those n numbers. This solution applies this
+observation to each column of the matrix, separately.
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2 M)$
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+typedef long long ll;
+#define all(x) (x).begin(), (x).end()
+#define rall(x) (x).rbegin(), (x).rend()
+using namespace std;
+void sol();
+
+int main() {
+	ios_base::sync_with_stdio(false);
+	cin.tie(NULL);
+	ll t;
```

Comment:
does everything need to be longlong

Reply:
I'm just trying to keep it as close to the original video as possible since all of his test cases passed in enough time.